assets/src/edit-story/components/panels/textStyle/padding.js

```diff
@@ -45,14 +45,27 @@ const Space = styled.div`
   flex: 0 0 10px;
 `;
 
-function PaddingControls({ selectedElements, pushUpdateForObject }) {
+function PaddingControls({
+  selectedElements,
+  pushUpdateForObject,
+  pushUpdate,
+}) {
   const padding = useCommonObjectValue(
     selectedElements,
     'padding',
     DEFAULT_PADDING
   );
+  const isPaddingLockOpen = getCommonValue(selectedElements, 'paddingLockOpen');
 
-  const [lockPaddingRatio, setLockPaddingRatio] = useState(true);
+  const [lockPaddingRatio, setLockPaddingRatio] = useState(
+    isPaddingLockOpen === MULTIPLE_VALUE ? true : !isPaddingLockOpen
+  );
+
+  useEffect(() => {
+    setLockPaddingRatio(
+      isPaddingLockOpen === MULTIPLE_VALUE ? true : !isPaddingLockOpen
```

Comment:
What's this doing? Setting it to the opposite if not multiple? Similarly above - why the `!`?

Reply:
Predefined in `isPaddingLock` getting from getCommonValue